src/common/components/search-input/search.css

```diff
@@ -0,0 +1,14 @@
+$border-color: $mid-grey;
+
+.searchInput {
+  background-image: url('https://assets.ubuntu.com/v1/4a72426e-search-icon-grey.svg');
+  background-position: right 4px bottom 2px;
```

Comment:
@bartaz I'm going to fix this in the next branch as I have all the functionality working now :)